frontend/src/locales/zh-CN.json

```diff
@@ -472,7 +474,9 @@
       "modal": {
         "resolve": "解决工单?",
         "cancel": "取消整个工单?",
-        "reopen": "重开工单?"
+        "reopen": "重开工单?",
+        "title": "{action}{type} [{name}]？",
```

Comment:
```suggestion
        "reopen": "重开工单？",
        "title": "{action}{type} [{name}]？",
```

Reply:
Should we use half-width character in Chinese?

Reply:
LGTM